src/plays/lofi-player/player.scss

```diff
@@ -0,0 +1,77 @@
+.player_container
```

Comment:
@riyajath-ahamed 

I see some CSS impacts like this

![image](https://user-images.githubusercontent.com/3633137/196456542-4717b073-495c-41f4-abaf-37f9b82fd160.png)

As a Thumb rule, always have a class name unique to ur play and use that as a prefix with all the style names. Can you please so that? I will test it after that. Thanks.

Reply:
Yes Done @atapas @koustov